### PR TITLE
Docs/users

### LINKED ISSUE #513: api-reference: Users - create a user

### PR DESCRIPTION
**Pull request by jstirnaman**
- Update descriptions for users. (Closes #513)
- Update user status description in schema.
- Trying out new permissions table format.
- Update a password (Closes https://github.com/influxdata/openapi/issues/511)
- Authorizations fixes
- Scripts fixes
- Linking

### CODE REVIEW

File: src/common/paths/users.yml [hunk 1]
```diff
@@ -72,11 +85,34 @@ post:
...
 
-    - `write-users`. Requires an InfluxDB API **Op** token.
+    | Action | Permission required | Restriction |
+    |:-------|:--------------------|:------------|
+    | Create a user | _[Operator token](https://docs.influxdata.com/influxdb/latest/security/tokens/#operator-token)_  | |
+
+    #### Related guides
+
+    - [Create a user](https://docs.influxdata.com/influxdb/latest/users/create-user/).
+    - [Create an API token scoped to a user](https://docs.influxdata.com/influxdb/latest/security/tokens/create-token/#create-a-token-scoped-to-a-user)
```
Comment (sunbryely-influxdata) [inline src/common/paths/users.yml:115]:
```suggestion
    - [Create an API token scoped to a user](https://docs.influxdata.com/influxdb/latest/security/tokens/create-token/#create-a-token-scoped-to-a-user).
```

File: src/common/paths/users_userID.yml [hunk 2]
```diff
@@ -4,17 +4,25 @@ get:
     - Security and access endpoints
     - Users
   summary: Retrieve a user
+  description: |
+    Retrieves a [user]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#user).
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/common/paths/users_userID.yml:12]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

File: src/cloud/paths/users.yml [hunk 3]
```diff
@@ -5,16 +5,81 @@ get:
...
+    | List all users | Operator token | InfluxData internal use only |
+    | List a specific user | `read-users` or `read-user USER_ID` |
+
+    Replace the following:
+
+    - `USER_ID`: ID of the user that you want to retrieve.
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/users.yml:30]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

File: src/cloud/paths/users.yml [hunk 4]
```diff
@@ -24,7 +89,25 @@ post:
...
+
+    #### Required permissions for InfluxDB Cloud
+
+    | Action | Permission required | Restriction |
+    |:-------|:--------------------|:------------|
+    | Create user | Operator token | InfluxData internal use only |
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/users.yml:110]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

File: src/cloud/paths/users_userID.yml [hunk 5]
```diff
@@ -3,17 +3,25 @@ get:
   tags:
     - Users
   summary: Retrieve a user
+  description: |
+    Retrieves a [user]({{% INFLUXDB_DOCS_URL %}}/reference/glossary/#user).
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/users_userID.yml:11]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

File: src/cloud/paths/users_userID.yml [hunk 6]
```diff
@@ -26,8 +34,27 @@ patch:
...
+
+    #### Required permissions for InfluxDB Cloud
+
+    | Action | Permission required | Restriction |
+    |:-------|:--------------------|:------------|
+    | Update user | Operator token | InfluxData internal use only |
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/users_userID.yml:55]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

File: src/cloud/paths/users_userID.yml [hunk 7]
```diff
@@ -56,17 +85,41 @@ delete:
...
+
+    #### Required permissions
+
+    | Action | Permission required | Restriction |
+    |:-------|:--------------------|:------------|
+    | Delete user | Operator token | InfluxData internal use only |
+
+    #### Related guides
+
+    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/)
```
Comment (sunbryely-influxdata) [inline src/cloud/paths/users_userID.yml:109]:
```suggestion
    - [Manage users]({{% INFLUXDB_DOCS_URL %}}/organizations/users/).
```

**Review by sunbryely-influxdata — APPROVED** [review]

File: src/common/paths/orgs_orgID.yml [hunk 8]
```diff
@@ -151,7 +151,7 @@ delete:
 
     #### Related guides
 
-    - [Delete organization]({{% INFLUXDB_DOCS_URL %}}/organizations/delete-orgs/)
+    - [Delete organizations]({{% INFLUXDB_DOCS_URL %}}/organizations/delete-orgs/)
```
Comment (jstirnaman) [inline src/common/paths/orgs_orgID.yml:154]: @sunbryely-influxdata This PR cleans up (deletes) periods for linked page titles in lists. For example...

Comment (jstirnaman) [reply src/common/paths/orgs_orgID.yml:154]: ...also whitespace

**Pull request merged**